Backend/src/main/java/idealab/api/repositories/EmployeeRepo.java

```diff
@@ -1,11 +1,22 @@
 package idealab.api.repositories;
 
-import idealab.api.model.Employee;
+import java.util.List;
+
+import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.repository.CrudRepository;
 
+import idealab.api.dto.response.BasicEmployeeResponse.EmployeeBasic;
+import idealab.api.model.Employee;
+
 public interface EmployeeRepo extends CrudRepository<Employee, Integer> {
     Employee findEmployeeById(Integer id);
     Employee findEmployeeByUsernameEquals(String username);
     Employee findEmployeeByUsername(String username);
+
+    /**
+     * Gets a list of all employees with first name, last name, username, and role.
+     */
+    @Query(value = "SELECT first_name as firstName, last_name as lastName, username, role from Employee", nativeQuery = true)
```

Comment:
Add constructor 
example:
`@Query("select new idealab.api.model.submodel.EmployeeBasic(e.firstName, e.lastName, e.username, e.role) from Employee e")`

Reply:
Hey so I think this still requires the EmployeeBasic class. Is there a benefit to doing it this way over the way I had it with the interface?